PlayCover/Views/Settings/UninstallSettings.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  UninstallSettings.swift
+//  PlayCover
+//
+//  Created by TheMoonThatRises on 9/26/22.
+//
+
+import SwiftUI
+
+class UninstallPreferences: NSObject, ObservableObject {
+    static var shared = UninstallPreferences()
+
+    @objc @AppStorage("ClearAppDataUninstall") var clearAppData = false
+    @objc @AppStorage("RemoveAppKeymapUninstall") var removeAppKeymap = false
+    @objc @AppStorage("RemoveAppSettingUninstall") var removeAppSettings = false
+    @objc @AppStorage("RemoveAppEntitlementsUninstall") var removeAppEntitlements = false
+
+    @AppStorage("ShowUninstallPopup") var showUninstallPopup = true
+}
+
+struct UninstallSettings: View {
+    public static var shared = UninstallSettings()
+
+    @ObservedObject var uninstallPreferences = UninstallPreferences.shared
+
+    @State private var showPruneFileAlert = false
+
+    var body: some View {
+        Form {
```

Comment:
Having "when uninstalling" on each line is redundant. It would be better to have something like this:
![Preview settings](https://user-images.githubusercontent.com/47154119/196594575-1abd2d76-ed07-4d08-b7f3-e11bd44cad59.png)
Saying perhaps `When uninstalling an app:`